src/app/DemoScreen.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+
+import { AUTH_ROUTER_PATHS } from 'features/auth/constants';
+import IsLoggedIn from 'features/auth/guards/IsLoggedIn';
+import { selectProfile } from 'common/services/user';
+import { AppState } from './store';
+
+/**
+ * Throw-away component
+ * This is just for demo purposes
+ */
+
+const mapStateToProps = (state: AppState) => ({
+  profile: selectProfile(state),
+});
+
+type Props = ReturnType<typeof mapStateToProps>;
+
+const DemoScreen = IsLoggedIn(({ profile }: Props) => (
+  <div>
+    ID: {profile!.id}
```

Comment:
toto neni cool, skor if profile ? null : screen

to by sme mali ! vsade, lebo kazda featura zacne s empty stavom, sice budu prazdne struktury ako array alebo obj, ale aj null moze byt

Reply:
jasne ze je to zle, to je len throwaway co najsimple component